### PR TITLE
Certified Connector First Pull

### PR DESCRIPTION
**Pull request by jpjsantos**
Custom Connector for assessment.

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Mobsim Send SMS/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+  {
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Chave de API",
+          "description": "O Chave de API para esta API",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Mobsim Send SMS/apiProperties.json:8]: @jpjsantos please make sure that all summaries, x-ms-summaries, and descriptions are in English and are grammatically correct and free of typographical errors. You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/sms": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json:26]: @jpjsantos empty response schemas are not allowed, please refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).

File: certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/sms": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json:25]: @jpjsantos it's not allowed to use default as the only response, please refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) when defining your responses.

File: certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,133 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MOBSIM Send SMS",
+    "description": "Communicate with your customers in an assertive and direct way using the \"MOBSIM - Send SMS\" Connector. SMS - with high delivery and opening rates - makes it possible to send promotional and transactional campaigns, alerts, reminders, notifications, invites and more, allowing for a complete experience with personalized messages and the convenience of scheduled deliveries to mobile phone users.",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json:5]: @jpjsantos please make the description more concise. It should describe the main purpose and value offered by your connector. More information can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

File: certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+    "version": "1.0",
+    "contact": {
+      "name": "Suporte Mobsim",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json:8]: @jpjsantos please rename this one to `Mobsim Support`.

File: certified-connectors/Mobsim Send SMS/README.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+## Conector Mobsim Send SMS
```
Comment (v-EgorKozhadei) [inline certified-connectors/Mobsim Send SMS/README.md:1]: @jpjsantos please format the readme.md file according to [this template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). Your readme should contain all the necessary headings.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @jpjsantos! Thank you for your PR! Please check all the comments after the review, thank you!

**Comment by jpjsantos** [comment]
All requested changes fulfilled. Waiting for review. 

Thanks.

File: certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/sms": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
```
Comment (v-EgorKozhadei) [inline certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json:26]: @jpjsantos please provide a schema for your 200 response if this is an action, not a trigger.

File: certified-connectors/Mobsim Send SMS/README.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
+# Conector Mobsim Send SMS
```
Comment (v-EgorKozhadei) [inline certified-connectors/Mobsim Send SMS/README.md:1]: @jpjsantos please remove connector from the title here.

**Comment by v-EgorKozhadei** [comment]
Hello @jpjsantos, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) using your latest commit id, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**